CODEOWNERS

```diff
@@ -238,10 +238,9 @@ k8s/**/common/ethos/identity.yaml @hmcts/ethos
 k8s/**/common/ethos/repl-docmosis-service.yaml @hmcts/ethos
 k8s/aat/common/ethos/ecm-consumer.yaml @hmcts/ethos
 
-### Bulk scan and print
+### Bulk scan
 k8s/namespaces/bsp/ @hmcts/bulk-scan
 k8s/**/common/bsp/identity.yaml @hmcts/bulk-scan
-k8s/**/common/bsp/bulk-scan-payment-processor.yaml @hmcts/bulk-scan
```

Comment:
any reason this is removed from codeowners ?

Reply:
everything has moved to `namespace/bsp` now

Reply:
It doesnt exist anymore. Also, u have full control on what files you add/edit in `k8s/namespaces/bsp/` , but u need approval when u add them to kustomizations.